cloud/cmd/inter_cluster_gateway/main.go

```diff
@@ -41,21 +43,27 @@ var (
 
 // initialize the commandline options
 func InitFlag() {
-	var logLevel string
-	flag.StringVar(&remoteIcgwIPstr, "remote_icgw", "", "the ip address of the remote Inter-Cluster Gateway")
-	flag.StringVar(&localDividerIPstr, "local_divider", "", "the ip address of the local divider")
-	flag.StringVar(&logLevel, "log_level", "3", "log level")
-
-	flag.Parse()
+	config, err := NewGatewayConfiguration("gateway_config.json")
+	if err != nil {
+		fmt.Printf("error setting gateway agent configuration: %v", err)
```

Comment:
use logger instead of fmt.

Reply:
The logger is not initialized at that time. I replaced it with panic as common practice.

Reply:
that works. thanks.